.gitignore

```diff
@@ -10,3 +10,7 @@
 
 # Configure files created by Vagrant and VirtualBox.
 /concent-builder-vm/.vagrant/
+/concent-vm/.vagrant/
+
+# Attached disk to vagrant
+/concent-vm/disk/
```

Comment:
Please create `.gitignore` inside the `concent-vm/` directory. I think that's better than putting everything into the global one.

Reply:
Done.